prime-router/src/main/kotlin/cli/ValidateUtilities.kt

```diff
@@ -0,0 +1,61 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktError
+import com.github.ajalt.mordant.rendering.TextColors.green
+import com.github.ajalt.mordant.rendering.TextColors.red
+import gov.cdc.prime.router.config.validation.ConfigurationType
+import gov.cdc.prime.router.config.validation.ConfigurationValidationFailure
+import gov.cdc.prime.router.config.validation.ConfigurationValidationResult
+import gov.cdc.prime.router.config.validation.ConfigurationValidationService
+import gov.cdc.prime.router.config.validation.ConfigurationValidationServiceImpl
+import gov.cdc.prime.router.config.validation.ConfigurationValidationSuccess
+import java.io.File
+
+class ValidateUtilities() {
+    companion object ValidateUtilities {
```

Comment:
rather than adding a companion object to a class, you can just make the object top level.

```kotlin
object ValidateUtilities {
  // ...
}
```